app/views/events/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= render 'layouts/navigation', current_user: @current_user%>
+
+<br><br>
+
+<table>
+    <thead>
+        <strong>PREVIOUS EVENTS</strong>
+    </thead>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <th>Location</th>
+            <th colspan="3">Date</th>
+            <th colspan="3">Creator</th>
+        </tr>
+
+        <% @events.past.each do |event| %>
```

Comment:
- [ ] Lets pass `@past_events` and `@future_events` from the controller instead of getting them from `@events` in the views.